### ISSUE TITLE
[feature request] Setting menu for screenlocker in session config

### ISSUE THREAD

**Issue by ringo32** [lxqt/lxqt#1664]
Just add a opion which to config preferended Locker, currently is xscreensaver standard.
in arch know of xdg-utils-slock can work around those but still  such fork is not fine to use, know lxqt using xdg-utils for that. is nicer to set up in de session-config

##### Expected Behavior

New lxqt should always point out as standard to xscreensaver in that setting.
But is always nice if you can select slock as example or lightdm-lock as example

--

if you change to slock you can setup  or something you can ush in menu to lock or as standard ctrl-alt-l with out re-edit the global keys

##### Current Behavior
if you select lock-screen it want xdg-screenlocker.

-

currently you copy the xdg-desktop-files to .local/share/applications and change something 
you change the lxqt-session --lock to slock  and global keys on slock works also this way

or using xdg-utils-slock from aur (arch)

##### Possible Solution

set it in session configuration to select prefered lock screen it works this way more integrated as select lock screen, as xfce use xflock4 script with some standards but is way to change if you want.. but in session-settings is way more integrated

**Comment by librewish** [comment]
if its going to be implemented then also add a way to set kscreenlocker as default lockscreen.
as when installing kwin its also installed as dependencie.

**Comment by ringo32** [comment]
yeah atleast you got option and less depended to xdg-utils. beside not WM agnostic , just choose several you can set locker by your self, but i dont know kscreenlocker, but  mostly like light-locker upcomming xfce4-screensaver (fork of mate-screensaver) or xscreensaver it add also something to startup to use the timing, but thats not very de job, for kscreenlocker should then only on the option ?  is there also a terminal command to call it ?
@librewish  ?

**Comment by Vladimir-csp** [comment]
If it is aware of logind stuff, then `loginctl lock-session $XDG_SESSION_ID` would do the trick.

**Comment by librewish** [comment]
no i dont know how to call kscreenlocker using terminal.
its kde desktop screen locker.
`kcmshell5 screenlocker`
only gives the setting window.
alt+ctrl+l is shortcut but it just invokes xscreensaver.

**Comment by n4nn31355** [comment]
Hey! Hope this time(oh, almost 2.5 years have passed) I will be more luky to merge this feature in lxqt, than trying to push it to xdg-utils =)

**Comment by n4nn31355** [comment]
Welcome to contribute, but not to get feedback, huh?

**Comment by stefonarch** [comment]
This got really forgotten somehow, sorry. 
Could you rebase it on current master?
